### PR TITLE
Add Scala 2.11.x build.

### PR DESCRIPTION
**Pull request by jsuereth**
This should not break the current pr validation, but enable us to try out 0.13.2 builds for the IDE.

Review by @gkossakowski @adriaanm and @dragos just to be sure.

### CODE REVIEW

File: sbt-0.13.x-on-2.10.x [hunk 1]
```diff
@@ -41,7 +41,7 @@
         uri:    "ivy:org.scalacheck#scalacheck_2.10;1.10.1"
       }, {
         name:   "sbinary",
-        uri:    "git://github.com/harrah/sbinary.git#v0.4.2"
+        uri:    "git://github.com/harrah/sbinary.git#master"
```
Comment (gkossakowski) [inline sbt-0.13.x-on-2.10.x:44]: Why that change?

File: sbt-0.13.x-on-2.11.x [hunk 2]
```diff
@@ -0,0 +1,114 @@
+# Warning: THIS FILE IS BEING USED FOR INTEGRATION WITH THE SCALA-IDE TEAM
+#          Pleases do not modify without contacting them.
+# Nighlty job: https://jenkins-dbuild.typesafe.com:8499/job/sbt-0.13.x-nightly-for-ide-on-scala-2.10.x
+{
+  properties: "file://"${PWD}"/versions.properties"
+  // Variables that may be external.  We have the defaults here.
+  vars: {
+    scala-version: "2.11.0-SNAPSHOT"
+    scala-version: ${?SCALA_VERSION}
+    publish-repo: "http://private-repo.typesafe.com/typesafe/ide-2.10"
```
Comment (gkossakowski) [inline sbt-0.13.x-on-2.11.x]: Shouldn't this be ide-2.11?

Comment (jsuereth) [inline sbt-0.13.x-on-2.11.x]: Good point!

File: sbt-0.13.x-on-2.10.x (hunk 1 above)
Comment (jsuereth) [inline sbt-0.13.x-on-2.10.x:44]: So they both line up on 0.4.3, or "to be released" 0.4.3 which works with 2.11.

**Comment by jsuereth** [comment]
updated, any remaining comments?

**Comment by adriaanm** [comment]
LGTM!

File: sbt-0.13.x-on-2.10.x (hunk 1 above)
Comment (gkossakowski) [inline sbt-0.13.x-on-2.10.x:44]: Ok. Do you plan to release 0.4.3 soon?

**Comment by gkossakowski** [comment]
LGTM.

File: sbt-0.13.x-on-2.10.x (hunk 1 above)
Comment (jsuereth) [inline sbt-0.13.x-on-2.10.x:44]: On the docket for tomorrow, methinks.

**Pull request merged**